safe/definitions/field_groups/field_groups.py

```diff
@@ -0,0 +1,140 @@
+# coding=utf-8
+
+"""Definitions relating to group of fields."""
+from safe.definitions.concepts import concepts
+from safe.definitions.field_groups.age_field_group import (
+    age_ratio_group,
+    age_count_group,
+    age_displaced_count_group)
+from safe.definitions.field_groups.age_vulnerability_field_group import (
+    age_vulnerability_ratio_group,
+    age_vulnerability_count_group,
+    age_vulnerability_displaced_count_group)
+from safe.definitions.field_groups.gender_vulnerability_field_group import (
+    gender_vulnerability_ratio_group,
+    gender_vulnerability_count_group,
+    gender_vulnerability_displaced_count_group)
+from safe.definitions.field_groups.disability_vulnerability_field_group import (
+    disability_vulnerability_ratio_group,
+    disability_vulnerability_count_group,
+    disability_vulnerability_displaced_count_group)
+from safe.definitions.field_groups.gender_field_group import (
+    gender_ratio_group,
+    gender_count_group,
+    gender_displaced_count_group)
+from safe.utilities.i18n import tr
+
+__copyright__ = "Copyright 2017, The InaSAFE Project"
+__license__ = "GPL version 3"
+__email__ = "info@inasafe.org"
+__revision__ = '$Format:%H$'
+
+aggregation_field_groups = [
+    age_ratio_group,
+    gender_ratio_group,
+    age_vulnerability_ratio_group,
+    gender_vulnerability_ratio_group,
+    disability_vulnerability_ratio_group
+]
+
+exposure_field_groups = [
```

Comment:
Hi @timlinux I know it's my fault, but I think we can change this to `population_field_groups` since it's only used by population exposure, not all exposure. Thanks